activiti-cloud-services-runtime-bundle/activiti-cloud-services-message-events/src/main/java/org/activiti/cloud/services/message/events/MessageEventsProcessEngineConfigurator.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright 2018 Alfresco, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.activiti.cloud.services.message.events;
+
+import org.activiti.engine.cfg.ProcessEngineConfigurator;
+import org.activiti.engine.impl.cfg.ProcessEngineConfigurationImpl;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.context.SmartLifecycle;
+
+public class MessageEventsProcessEngineConfigurator  implements ProcessEngineConfigurator, SmartLifecycle {
```

Comment:
I didn't get the purpose of this class. It seems to be doing some logging only.

Reply:
This is obsolete. I have removed it.